docs/WALLET_SDK.md

```diff
@@ -0,0 +1,131 @@
+# Fuel Wallet SDK
+
+- [Fuel Wallet SDK](#fuel-wallet-sdk)
+  - [How to use?](#how-to-use)
+  - [Quickstart](#quickstart)
+    - [Request connection](#request-connection)
+    - [List user accounts](#list-user-accounts)
+    - [Signing a message](#signing-a-message)
+  - [SDK API](#sdk-api)
+    - [Methods](#methods)
+      - [Connect](#connect)
+      - [Disconnect](#disconnect)
+      - [List Accounts](#list-accounts)
+      - [Request signature message](#request-signature-message)
+    - [Events](#events)
+      - [Accounts](#accounts)
+      - [Connection](#connection)
+
+## How to use?
+
+If the Fuel Wallet extension corrrectly installed on the user's browser, the Wallet SDK will be inject automatically on the
+`window` object on property `FuelWeb3`. To access it you can use `window.FuelWeb3`.
+
+```ts
+window.FuelWeb3.connect();
+```
+
+You can try this code directly on the developer console.
+
+## Quickstart
+
+### Request connection
+
+First of all, you need to connect and authorized your application, this will authorize your application to execute other actions.
+This can be done by acessing `FuelWeb3.connect()`.
```

Comment:
```suggestion
First of all, you need to connect and authorize your application; this will authorize your application to execute other actions. You can do this by accessing `FuelWeb3.connect()`.
```